README.md

```diff
@@ -130,12 +130,11 @@ issues.
 
 Copyright 2017 OpenMarket Ltd
 Copyright 2017 Vector Creations Ltd
-Copyright 2017-2025 New Vector Ltd 
```

Comment:
All changes below here were automatically done by my editor which is configured to remove trailing whitespace. I can undo these specific changes if requested.